packages/test-helper-automation/src/wd/print-logs.js

```diff
@@ -0,0 +1,31 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+/* eslint max-nested-callbacks: [0] */
+/* eslint no-console: [0] */
+
+'use strict';
+
+var wd = require('wd');
+
+wd.addPromiseChainMethod('printLogs', function printLogs() {
+  return this
+    .log('browser')
+    .then(function doPrintLogs(logs) {
+      logs.forEach(function formatLog(log) {
+        try {
+          log.message = JSON.parse(log.message);
+          const method = console[log.message.message.level] || console.log;
+          method.call(console, 'browser log:', log.message.message.text);
+        }
+        catch (err) {
+          console.log('browser log:', log.message);
```

Comment:
Should we print the error here? I guess this is just falling back if the attempt above doesn't work. Maybe we don't care why it failed - would just pollute the logs to print it out.